README.md

```diff
@@ -1,44 +1,42 @@
 # HOHQMesh.jl
 
-<!-- [![Docs-stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://trixi-framework.github.io/HOHQMesh.jl/stable) -->
+[![Docs-stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://trixi-framework.github.io/HOHQMesh.jl/stable)
 [![Docs-dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://trixi-framework.github.io/HOHQMesh.jl/dev)
 [![Build Status](https://github.com/trixi-framework/HOHQMesh.jl/workflows/CI/badge.svg)](https://github.com/trixi-framework/HOHQMesh.jl/actions?query=workflow%3ACI)
 [![Coveralls](https://coveralls.io/repos/github/trixi-framework/HOHQMesh.jl/badge.svg?branch=main)](https://coveralls.io/github/trixi-framework/HOHQMesh.jl?branch=main)
 [![License: MIT](https://img.shields.io/badge/License-MIT-success.svg)](https://opensource.org/licenses/MIT)
 
 This package is a thin Julia wrapper around the *High Order Hex-Quad Mesher*
-(a.k.a. **HOHQMesh**) created and developed by
+(a.k.a. [**HOHQMesh**](https://github.com/trixi-framework/HOHQMesh)) created and developed by
 [David A. Kopriva](https://www.math.fsu.edu/~kopriva/).
 HOHQMesh.jl is available on Linux, MacOS, and Windows.
 
-**Note: This package is currently _highly_ experimental!**
 
+## Installation
+If you have not yet installed Julia, please [follow the instructions for your
+operating system](https://julialang.org/downloads/platform/). HOHQMesh.jl works
+with Julia v1.6.
 
-## Getting started
-
-This package is still in proof-of-concept stage. Thus many things will not work
-as convenient as they do for properly registered packages.
-
-To install HOHQMesh.jl, you need to manually install it *and its dependency*
-[HOHQMesh_jll](https://github.com/trixi-framework/HOHQMESH_jll.jl)
-by executing the following lines in your Julia REPL:
+HOHQMesh.jl is a registered Julia package. Hence, you can install it by executing
+the following commands in the Julia REPL:
 ```julia
-julia> import Pkg
-
-julia> Pkg.add(url="https://github.com/trixi-framework/HOHQMESH_jll.jl")
-
-julia> Pkg.add(url="https://github.com/trixi-framework/HOHQMESH.jl")
+julia> import Pkg; Pkg.add("HOHQMesh")
 ```
+HOHQMesh.jl depends on the binary distribution of the
+[HOHQMesh](https://github.com/trixi-framework/HOHQMesh)
+mesh generator, which is available via the Julia package `HOHQMesh_jll.jl` and
```

Comment:
Add a link to the jll package?

Reply:
What would be the advantage? I don't typically link to the JLL package repo, since it usually does not convey any useful additional information.

Reply:
Fixed in 3452146.